controllers/object_controls.go

```diff
@@ -1298,6 +1380,35 @@ func TransformDevicePlugin(obj *appsv1.DaemonSet, config *gpuv1.ClusterPolicySpe
 		setContainerEnv(&(obj.Spec.Template.Spec.Containers[0]), MPSRootEnvName, config.DevicePlugin.MPS.Root)
 	}
 
+	// Update the driver-root volume if a custom driver install directory is configured with the operator
+	if config.HostPaths.DriverInstallDir != "" && config.HostPaths.DriverInstallDir != DefaultDriverInstallDir {
+		for index, volume := range obj.Spec.Template.Spec.Volumes {
+			if volume.Name == "driver-root" {
+				obj.Spec.Template.Spec.Volumes[index].HostPath.Path = config.HostPaths.DriverInstallDir
+			}
+		}
+	}
+
+	// If a custom devPath is configured which is not equal to 'hostRoot/dev' or 'driverInstallDir/dev', make it available
```

Comment:
Are we duplicating the logic between here and the toolkit (and the mig-manager?). Is this something that is applicable to all three cases?

Reply:
I have removed devPath from the ClusterPolicy API now. Consequently, this code has been removed.